packages/flutter_tools/lib/src/fuchsia/fuchsia_sdk.dart

```diff
@@ -156,4 +156,10 @@ class FuchsiaArtifacts {
 
   /// The pm tool.
   final File pm;
+
+  /// Returns true if the [sshConfig] file is not null and exists.
+  bool validateSshConfig() {
```

Comment:
A getter like the following would probably be more idiomatic:
```dart
bool get hasSshConfig => sshConfig != null && sshConfig.existsSync();
```
(or `hasValidSshConfig`)

Reply:
Done.  At one point this was a method that was also checking that pm and dev_finder were executable, which seemed heavy for a getter, but also turned out to be less helpful than I thought.